src/gesture.js

```diff
@@ -409,7 +409,7 @@ export class Gestures {
         'Recognizer is not currently allowed: %s', recognizer.getType());
     const overserver = this.overservers_[recognizer.getType()];
     if (overserver) {
-      overserver.fire(new Gesture(recognizer.getType(), data, Date.now(),
+      overserver.fire(new Gesture(recognizer.getType(), data, new Date(),
```

Comment:
This is a different API, and has perf implications. `Date` is much more expensive to create than the timestamp. If no one is actually using this, let's just delete it instead.